README.md

```diff
@@ -751,6 +751,37 @@ For supported properties, see
 [StreamClientConfiguration](https://fauna.github.io/fauna-js/latest/types/StreamClientConfiguration.html)
 in the API reference.
 
+## Debug logging
+
+To enable or disable debug logging, set the `FAUNA_DEBUG` environment variable
+to a string-encoded
+[`LOG_LEVELS`](https://fauna.github.io/fauna-js/latest/variables/LOG_LEVELS.html)
+integer:
+
+```shell
+# Enable logging for warnings (3) and above:
+export FAUNA_DEBUG="3"
+
+# Disable logging:
+export FAUNA_DEBUG="6"
```

Comment:
Logging is already disabled by default (set to level 6). I think this might suggest that folks _have_ to use this to disable. Thoughts?

Reply:
Good point. Removed the example and noted that debug logging is disabled by default here: [`323983d` (#312)](https://github.com/fauna/fauna-js/pull/312/commits/323983dde1eb5cff50dfbb9c7a744cbbcabae0bf)

Reply:
Actually... we say that it's disabled later. I'll just remove the example.